depfinder/main.py

```diff
@@ -260,7 +260,7 @@ def parse_file(python_file):
         PACKAGE_NAME = os.path.basename(python_file).split('.')[0]
         logger.debug("Setting PACKAGE_NAME global variable to {}"
                      "".format(PACKAGE_NAME))
-    with open(python_file, 'r') as f:
+    with open(python_file, 'r', encoding='utf-8-sig') as f:
```

Comment:
Is doing this for _all_ files a good idea?

Reply:
It seemed safe for the files I tried to run it on, but I'm not certain.

Reply:
I'd be interested in two possible paths here. One hacky and one less so. The hacky way would be to just catch the exception that's being raised by the adal code, set the encoding and open it again. THe other way would be to use something like python-magic or chardet to attempt to automatically detect the encoding and use it to open the file. I'd be fine with just a try/except to fix this adal problem

Reply:
I've pushed up the try except and it worked on my adal test